src/include/mpi.h.in

```diff
@@ -775,8 +775,8 @@ extern MPI_F08_status *MPI_F08_STATUSES_IGNORE MPICH_API_PUBLIC;
 typedef int (MPI_Grequest_cancel_function)(void *, int); 
 typedef int (MPI_Grequest_free_function)(void *); 
 typedef int (MPI_Grequest_query_function)(void *, MPI_Status *); 
-typedef int (MPIX_Grequest_poll_function)(void *, MPI_Status *);
-typedef int (MPIX_Grequest_wait_function)(int, void **, double, MPI_Status *);
```

Comment:
Let's drop this patch. I don't think it really buys us anything in this series of commits.

Reply:
The reason I made this is for a potential future patch, we could record those function callback in the progress hook, and make progress for them from all threads. In that case, we don't have the status to record. I could reintroduce this on that one though (or use MPI_STATUS_IGNORE)

Reply:
How "potential" is that future patch? :) Like, is it already in the works and will be submitted with the MDTA stuff?

Reply:
Yes, I have that locally, plan to push it together with the MDTA stuff once this is cleaned up. Or it can be a separate PR.